pages/index.js

```diff
@@ -189,9 +188,11 @@ export default function Home() {
               Blockchain statistics for $IRON
             </Text>
             <HStack>
-              <Button variant="secondary" size="medium">
-                View All Blocks
-              </Button>
+              <Link href="/explorer" passHref>
```

Comment:
Please update links using enum added with Breadcrumbs, I added those when Breadcrumbs wasn't merged.
Also please check all block links in header and footer.